src/empire/city.c

```diff
@@ -581,7 +603,7 @@ static void set_new_monument_elements_production(int empire_id, empire_city *cit
                     if (city->is_sea_trade || (empire_id == damascus_empire_id && city->name_id == 12)) {
                         change_selling_of_resource(city, RESOURCE_STONE, NOT_SELLING);
```

Comment:
This function (and its definition above) should go away, right?

All the merges made this return.

Reply:
Apologies, I thought I was careful with checking all the changes in the merger. Clearly missed this.
Fixed.